WeakAurasOptions/OptionsFrames/OptionsFrame.lua

```diff
@@ -166,11 +166,11 @@ function OptionsPrivate.CreateFrame()
   end
 
   if not (xOffset and yOffset) then
-    xOffset = (defaultWidth - GetScreenWidth()) / 2
-    yOffset = (defaultHeight - GetScreenHeight()) / 2
+    xOffset = GetScreenWidth() / 2
+    yOffset = GetScreenHeight() - defaultHeight / 2
   end
 
-  frame:SetPoint("TOPRIGHT", UIParent, "TOPRIGHT", xOffset, yOffset)
+  frame:SetPoint("TOP", UIParent, "BOTTOMLEFT", xOffset, yOffset)
```

Comment:
Why do you want to change this? This invalidates the stored settings.

Reply:
Because every coordinate you get is from bottomleft corner of the screen, and in order to use the opposite corner, you have to calculate where it is. With this there is no offsetting the coordinates based on GetScreenWidth (or height), so even if someone overwrites the functions, this will still work. (Overwriting GetScreenWidth is something that i have seen some people do in attempt to fix their ultrawide screen UIs)